assets/js/components/SaptuneDetails/SaptuneDetails.test.jsx

```diff
@@ -110,4 +110,54 @@ describe('SaptuneDetails', () => {
       staging.solutions_ids
     );
   });
+
+  it.each([
+    { solution: null },
+    { solution: { notes: [] } },
+    { solution: undefined },
+  ])(
+    'should display dashes for "Enabled Solution," "Applied Solution," "Staged Notes," and "Staged Solutions" when the solution is null, and staging is disabled',
+
+    ({ solution }) => {
+      const staging = { enabled: false, notes: [], solutions_ids: [] };
+
+      const {
+        configured_version: configuredVersion,
+        package_version: packageVersion,
+        services,
+
+        tuning_state: tuningState,
+      } = saptuneStatusFactory.build();
+      const { hostname, id: hostID } = hostFactory.build();
+      renderWithRouter(
+        <SaptuneDetails
+          appliedSolution={solution}
+          enabledSolution={solution}
+          hostname={hostname}
+          hostID={hostID}
+          configuredVersion={configuredVersion}
```

Comment:
Send `enabledNotes` and `appliedNotes` as empty lists and check we display them as "empty"